draft-ietf-rtcweb-security-arch.xml

```diff
@@ -1299,14 +1299,13 @@ a=sendrecv
               </preamble>
               <artwork type="inline"><![CDATA[
 identity-attribute  = "identity:" identity-assertion
-                      [ SP identity-extension
-                        *(";" [ SP ] identity-extension) ]
+                        *(SP identity-extension)
```

Comment:
Doesn't this lose the semicolon?

Reply:
My comment was that we should use semicolon OR space as separator. Personally I would prefer semicolon, but I think Martin earlier indicated that current implementations use space.